fetcher/src/job/job.api.ts

```diff
@@ -113,6 +113,24 @@ export async function fetchDataFeed({
   }
 }
 
+export async function fetchDataFeedByAggregatorId({
+  aggregatorId,
+  logger
+}: {
+  aggregatorId: bigint
+  logger: Logger
+}): Promise<{ timestamp: string; value: bigint }> {
```

Comment:
We could reuse this type :) 
```suggestion
}): Promise<IAggregate> {
```